### PR TITLE
Random Battle: Update Swellow, Flareon

### PR DESCRIPTION
**Pull request by ascriptmaster**
Remove Endeavor.

Force Guts on Pokemon with the ability and have Protect (but not Wish, for
the occasional Flareon)

### CODE REVIEW

File: data/scripts.js [hunk 1]
```diff
@@ -1700,7 +1700,7 @@ exports.BattleScripts = {
 			if (abilities.indexOf('Chlorophyll') >= 0 && ability !== 'Solar Power' && hasMove['sunnyday']) {
 				ability = 'Chlorophyll';
 			}
-			if (abilities.indexOf('Guts') >= 0 && ability !== 'Quick Feet' && (hasMove['facade'] || (hasMove['rest'] && hasMove['sleeptalk']))) {
+			if (abilities.indexOf('Guts') >= 0 && ability !== 'Quick Feet' && (hasMove['facade'] || (hasMove['protect'] || !hasMove['wish']) || (hasMove['rest'] && hasMove['sleeptalk']))) {
```
Comment (TheImmortal) [inline data/scripts.js]: I think you meant `&&`

**Comment by TheImmortal** [comment]
Speaking of Flareon, should it have Wish/Protect? It doesn't really work. I think we should just replace it with Rest/Talk.

**Comment by ascriptmaster** [comment]
As much as I'd like to say that it should keep it because they have 25% usage on the PS! stats, at the same time Flareon's usage stats seem to slightly favor Flash Fire, while random battles will probably prefer Guts+Toxic Orb?

**Comment by TheImmortal** [comment]
Cause Flareon has a Special Defensive Smogon set, with Lava Plume even. Mixing that with Guts set doesn't work. I can't say I've ever gotten a good Flareon, nor have I seen one.

I say we give it `["flareblitz","facade","superpower","flamecharge","rest","sleeptalk"]`

**Comment by ascriptmaster** [comment]
Maybe I should remove the Wish check then, hmm

**Comment by TheImmortal** [comment]
Indeed you should

**Comment by ascriptmaster** [comment]
OK.

They're still 2 commits because whatever

**Pull request merged**